Makefile

```diff
@@ -33,9 +33,9 @@ run-integration-tests:
 	$(MAKE) docker
 	$(MAKE) -C test/services docker-build docker-push
 	$(MAKE) cluster-prepare-wait
-	$(MAKE) configure-vault
 	$(MAKE) -C charts m4d
 	$(MAKE) -C manager wait_for_manager
+	$(MAKE) configure-vault
```

Comment:
Changes here need to be reflected in the web site contributing page as well.

Reply:
Thanks will update the web page as well then.